index.html

```diff
@@ -90,14 +90,15 @@
 		<span id="electronDragZone" style="pointer-events: none; z-index:-10; position:absolute;top:0;left:0;width:100%;height:2%;-webkit-app-region: drag;min-height:20px;"></span>
 		<div id="header">
 			
-			<a id="logoname" href="./" style="text-decoration: none; color: white; margin: 0 2px 0px 8px;">
-				<span data-translate="logo-header">
-					<span id="qos">V</span>DO.Ninja 
```

Comment:
"qos" changes color to red, when there is a loss of Internet, so this conflicts with an existing feature.

https://vdo.ninja/media/logo_cropped_512.png is a large image; it will slow down page load needlessly. Any non-essential extra image requests I try to avoid, loading a few icons on page load inline as base64 data rather than external links.

Reply:
It might be possible to use an inline SVG of the VDO.Ninja logo, ~ 1 to 3 kb in size, and have the entire SVG change to red when there is a loss of Internet.  I can probably update the qos code, adding and removing a class to the SVG; i"d just need a class available that changes its color.

Reply:
I'd personally prefer the logo to also remain "VDO.NInja" (stylized or not), rather than the ninja face, as I feel a the name is important to the branding.

Reply:
Thank you for your quick Feedback!

 In 4c4d58c I removed the image and put back your logoname with the offline functionality. I wanted to add some styling to `#logoname` but when I tested it locally, the logo did not show up for me (not even red when there is "no internet" as you mentioned). Maybe you can have a look into it and check how the logo looks now, as I am not sure how it is currently displayed.

Reply:
Also I might have screwed up the css formatting, because of my own formatter, is there any formatter I am supposed to use for this project?

Reply:
I've now synced my CSS changes with GIthub, but I'll likely revert both yours and/or my change after some more testing.  

Changing the header height has large implications, so this isn't really something I think I can keep for long, but I pushed it to alpha to give myself some time to reflect on the change.

![image](https://github.com/steveseguin/vdo.ninja/assets/2575698/118ce9f8-2e26-4bd4-9cc2-c899adeaceb2)

I'd probably suggest avoiding changing the CSS or style of vdo.ninja itself, as there are a lot more ramifications for any change than how it might seem.  

As for the CSS file formatting, a space counts as a byte, so two spaces is two bytes, while a tab counts as one byte I believe.  So I generally try to use tabs to reduce space, although the CDN I use will often compress things down anyways -- but it's not always on or do all users have a CDN.  I may end up reverting the CSS formatting as a result

Reply:
There are sample apps, code, and such here:  https://github.com/steveseguin/vdo.ninja/tree/develop/examples  

Many of these are not documented or styled;  you'd have a lot of freedom to style pretty much any of these apps / pages however you'd want, without my push back from me.

I just would preface by saying I try to avoid any external dependency where possible, and due to their nature of being examples, the code / framework should ideally be kept simple.

Reply:
Totally understandable, if you want to revert. I already had the feeling that there will be reasons why things look how they look and that I should not try to change it up, since it might not be performant enough / has other consequences.

I will look into the example apps / translations in the near future and try to make a contribution there! Thanks for giving it a look nevertheless 👍